README.md

```diff
@@ -1,4 +1,4 @@
-## Description
+### Description
```

Comment:
Why change the structure of the document? I don't think it is right to start a readme with a level 3 heading

Reply:
Updated